apps/browser-extension-wallet/sentry.js

```diff
@@ -2,6 +2,76 @@ import * as Sentry from '@sentry/react';
 import { v4 as uuidv4 } from 'uuid';
 import { storage } from 'webextension-polyfill';
 
+const STANDARD_QUOTA_LIMIT = 10 * 1024 * 1024; // ~10MB
+const NEAR_QUOTA_THRESHOLD = STANDARD_QUOTA_LIMIT * 0.8; // 80% of quota
+
+/**
+ * Checks storage quota asynchronously when a LevelDB error occurs
+ * This runs independently of error handling to avoid blocking
+ */
+const checkStorageQuota = async () => {
+  try {
+    const bytesInUse = await storage.local.getBytesInUse(null);
+    const hasUnlimitedStorage = typeof storage?.local?.QUOTA_BYTES === 'undefined';
```

Comment:
```suggestion
    const hasUnlimitedStorage = storage.local.QUOTA_BYTES === undefined;
```
Unnecessary optional chaining since storage is imported